app/views/organizations/adopter_fosterer/adopted_pets/files/index.html.erb

```diff
@@ -1,20 +1 @@
-<turbo-frame id="pet_files">
-  <ul class="nav nav-pills mb-3" id="pills-tab" role="tablist">
-    <li class="nav-item" role="presentation">
-      <button class="nav-link active" id="pills-files-tab" data-bs-toggle="pill" data-bs-target="#pills-files" type="button" role="tab" aria-controls="pills-files" aria-selected="true"><%= t('general.files') %></button>
-    </li>
-    <li class="nav-item" role="presentation">
-      <button class="nav-link" id="pills-tasks-tab" data-bs-toggle="pill" data-bs-target="#pills-tasks" type="button" role="tab" aria-controls="pills-tasks" aria-selected="false"><%= t('general.tasks') %></button>
-    </li>
-  </ul>
-
-  <div class="tab-content" id="pills-tabContent">
-    <div class="tab-pane fade show active" id="pills-files" role="tabpanel" aria-labelledby="pills-files-tab">
-      <%= render "organizations/shared/file_attachment_table",
-                  locals: {pet: @pet} %>
-    </div>
-    <div class="tab-pane fade" id="pills-tasks" role="tabpanel" aria-labelledby="pills-tasks-tab">
-      <%= turbo_frame_tag "pet_tasks", src: adopter_fosterer_adopted_pet_tasks_path(@pet), loading: :lazy %>
-    </div>
-  </div>
-</turbo-frame>
+<%= render "organizations/adopter_fosterer/shared/pet_files_table", pet_tasks_path: adopter_fosterer_adopted_pet_tasks_path(@pet, pet_id: @pet.id) %>
```

Comment:
I take it we have to explicitly pass the `pet_id` here as it is not being made available by passing `@pet1 by itself?

Reply:
Thank you @kasugaijin for feedback. I did thought on this thing of passing additional parameter but was needed to get common param to be used in` file_base_controller` file.

Without explicitly passing `pet_id`, the system defaults to `adopted_pet_id` and `fostered_pet_id` params respectively. This creates inconsistency in parameter naming when these routes share a common `file_base_controller`.